packages/rfw/test/runtime_test.dart

```diff
@@ -244,7 +244,7 @@ void main() {
       ),
     );
     expect(tester.takeException().toString(), contains('Could not find remote widget named'));
-    expect(tester.widget<ErrorWidget>(find.byType(ErrorWidget)).message, 'Could not find remote widget named widget in a, possibly because some dependencies were missing: b');
+    expect(tester.widget<ErrorWidget>(find.byType(ErrorWidget)).message, contains('Could not find remote widget named widget in a, possibly because some dependencies were missing: b'));
```

Comment:
why "contains"? what else does it contain?

Reply:
The `_defaultErrorWidgetBuilder` also adds "\nSee also: https://flutter.dev/docs/testing/errors" to the error message.

Reply:
I see. Looks like that's only included in debug builds. That's probably fine.